protocol/messenger_base_test.go

```diff
@@ -19,6 +19,7 @@ import (
 const DefaultProfileDisplayName = ""
 
 func (s *MessengerBaseTestSuite) SetupTest() {
+	s.tmpdir = s.T().TempDir()
```

Comment:
We only seem to need it in `TestBackupContactsLocally`, maybe create it right there only?